src/app/components/calendar/calendarForm.tsx

```diff
@@ -36,17 +32,18 @@ const CalendarForm = () => {
   const [calendarSetup, setCalendar] = useLocalStorage('calendarData', calendarDefaultValues)
   const [calendarDates, setCalendarDates] = useLocalStorage('calendarDatesData', calendarDatesDefault)
 
+  // https://dev.to/gvegacl/nextjs-server-prop-did-not-match-client-prop-and-conditional-styleclass-rendering-2i4c
   const [mount, setMount]= useState(false)
```

Comment:
Just FYI: I just learned that this issue may be later solved with [partial prerendering](https://nextjs.org/docs/app/api-reference/next-config-js/partial-prerendering)

The server cannot know the state of client's local storage, so it cannot render the right calendar. With partial prerendering, it ought to be possible to exclude the calendar from rendering on server, but still render it immediately on the client